service/src/integration-test/resources/json/expectedResult.json

```diff
@@ -7,3785 +7,2820 @@
     "value": "2"
   },
   "type": "message",
-  "entry": [
```

Comment:
Can you please go back to previous style of braces and brackets? It's really hard to see what really changed here. The same with expectedResult2.json